runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,155 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+            "TERM=xterm"
```

Comment:
Do we need to set `TERM`? Hopefully not as that's something that's can vary widely from host to host.

Reply:
There is no host-wide TERM. It should be used by a user who are connecting to the host.